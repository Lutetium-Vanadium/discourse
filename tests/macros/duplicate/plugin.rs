fn main() {
    let q = inquisition::questions![plugin {
        plugin: todo!(),
        plugin: todo!(),
    }];
}
