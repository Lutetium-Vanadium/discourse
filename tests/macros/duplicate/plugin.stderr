error: duplicate option `plugin`
 --> $DIR/plugin.rs:4:9
  |
4 |         plugin: todo!(),
  |         ^^^^^^
